src/api/app/models/bs_request.rb

```diff
@@ -150,11 +150,7 @@ def self.actions_summary(payload)
     ret.join(', ')
   end
 
-  def self.new_from_xml(xml)
-    hashed = Xmlhash.parse(xml)
-
-    raise SaveError, 'Failed parsing the request xml' unless hashed
-
+  def self.new_from_xml(hashed)
```

Comment:
The function name does not what it says anymore, it is more like ``new_from_hash``, right?

Reply:
@ChrisBr:

> The function name does not what it says anymore, it is more like new_from_hash, right?

Yes. Changed.